### DISCUSSION TITLE
Pass paramaters as windows

### DISCUSSION THREAD

**Discussion by AloneAndUnknown** [yuezk/GlobalProtect-openconnect#202] [General]
First. Thank you! I bought you a few cups of coffee. 

Looking to pass a few parameters to the server. I noticed in the GUI you had clientos, but i was also looking for the clientosversion as thats still being pass as ubuntu 16 or linux-64. Where can we edit that field? thanks!

**Comment by yuezk** [comment]
Thanks for your support!

I want to make sure which request you want to carry the `clienosversion` parameter. Did you mean the `clientVer` in the following request?

```
https://vpn.xxx.com/global-protect/prelogin.esp?tmp=tmp&kerberos-support=yes&ipv6-support=yes&clientVer=4100&clientos=Linux
```

**Comment by AloneAndUnknown** [comment]
Yes i believe it is here.

"direct=yes&clientVer=4100&os-version=Ubuntu 20.04.5 LTS&clientos=Linux&portal-prelogonuserauthcookie="

the OS version appears to be populating and conflicting if i change the clientos to windows.

**Comment by yuezk** [comment]
Are you installing it from the source code? If yes, you can try it with the latest code of the master branch. Otherwise, I will release a new version later.

**Reply by AloneAndUnknown** [reply]
Nah I'm lazy and installed it through the apt repo. Ill wait until the next revision ;) ty for addressing it though!

**Reply by yuezk** [reply]
@AloneAndUnknown released in 1.4.9

**Reply by AloneAndUnknown** [reply]
Works good now to where we have cleaned out the Ubuntu version but somewhere it is passing "Linux-64". Possibly where its passing the Client version. 

  | client_os | Windows |  
  | client_os_ver | linux-64 |  
  | client_ver | 4.0.5

**Reply by yuezk** [reply]
I will make all these parameters configurable.
